<launch>
  <rosparam command="load" file="$(find simtrack_nodes)/config/pr2_camera_topics.yaml"/>
  <rosparam command="load" file="$(find simtrack_nodes)/config/parameters.yaml"/>
  <param name="simtrack/use_compressed_streams" value="true" />
  <param name="simtrack/tracker/device_id" value="1" />
  <param name="simtrack/detector/device_id" value="0" />

  <rosparam command="load" file="$(find simtrack_nodes)/config/demo_objects.yaml"/>
  <param name="simtrack/model_path" value="$(find simtrack_nodes)/../data/object_models"/>

  <node name="simtrack" pkg="simtrack_nodes" type="pr2_cam_switch_node" output="screen"/>
</launch>
